test-data/unit/check-ignore.test

```diff
@@ -217,7 +217,7 @@ def f() -> None: pass
 [out]
 
 [case testCannotIgnoreBlockingError]
-yield  # type: ignore  # E: "yield" outside function
+yield  # type: ignore  # E: 'yield' outside function
```

Comment:
We're actually trying to standardize on double quotes in error messages. I believe it gets highlighted better.

Reply:
OK, changed back to double quotes